package.json

```diff
@@ -56,7 +56,7 @@
     "mkdirp": "^0.5.1",
     "mocha": "^5.0.1",
     "mock-fs": "^4.4.1",
-    "prettier": "^1.13.3",
+    "prettier": "^1.15.1",
```

Comment:
looks like we could also bump `eslint-config-prettier` to 3.1.0

Reply:
done!